src/doc/style-guide/src/statements.md

```diff
@@ -159,8 +159,8 @@ before the `else`.
 
 If the initializer expression is multi-line, the `else` keyword and opening
 brace of the block (i.e. `else {`) should be put on the same line as the end of
-the initializer expression, with a space between them, if all the following are
-true:
+the initializer expression, with a space between them, if and only if all the
+following are true:
```

Comment:
I feel like the ambiguity comes from the fact that this all starts with the "if the initializer expression multi-line" up on line 160, as that could be read as the following text only being applicable if that also holds.

I'll think on some additional text proposals, but wanted to go ahead and share that.

Reply:
That's true, but in the single-line case it doesn't matter if you try to apply these rules or not; the three conditions can't possibly hold for a single-line initializer. You can't have a single-line initializer expression that consists of nothing but closing parentheses, square brackets, and/or braces.